Source/Core/UICommon/GameFile.cpp

```diff
@@ -547,6 +549,106 @@ std::vector<DiscIO::Language> GameFile::GetLanguages() const
   return languages;
 }
 
+bool GameFile::CheckIfTwoDiscGame(const std::string& game_id) const
+{
+  static constexpr auto two_disc_game_ids = {
+      // Baten Kaitos: Eternal Wings and the Lost Ocean
+      "GKBPAF",
+      "GKBEAF",
+      "GKBJAF",
+      // Baten Kaitos Origins
+      "GK4E01",
+      "GK4J01",
+      // Conan
+      "GC9P6S",
+      // Dragon Quest X: Mezameshi Itsutsu no Shuzoku Online
+      "S4MJGD",
+      "S4SJGD",
+      "S6TJGD",
+      "SDQJGD",
+      // Enter the Matrix
+      "GMXP70",
+      "GMXJB2",
+      "GMXE70",
+      // Freaky Flyers
+      "GFFE5D",
+      // GoldenEye: Rogue Agent
+      "GOYP69",
+      "GGIJ13",
+      "GOYE69",
+      "GOYD69",
+      "GOYF69",
+      "GOYS69",
+      // Killer7
+      "GK7E08",
+      "GK7P08",
+      "GK7J08",
+      // Lupin the 3rd: Lost Treasure by the Sea
+      "GL3JE8",
+      // Medal of Honor: Rising Sun
+      "GR8F69",
+      "GRZJ13",
+      "GR8E69",
+      "GR8P69",
+      "GR8D69",
+      // Metal Gear Solid: The Twin Snakes
+      "GGSPA4",
+      "GGSJA4",
+      "GGSEA4",
+      // Resident Evil
+      "GBIP08",
+      "GBIE08",
+      "GBIJ08",
+      "DBJJ08",
+      // Resident Evil 4
+      "G4BE08",
+      "G4BP08",
+      "G4BJ08",
+      // Resident Evil Code: Veronica X
+      "GCDP08",
+      "GCDE08",
+      "GCDJ08",
+      // Resident Evil Zero
+      "GBZE08",
+      "GBZJ08",
+      "GBZP08",
+      // Tales of Symphonia
+      "GQSDAF",
+      "GQSFAF",
+      "GQSEAF",
+      "GQSPAF",
+      "GQSIAF",
+      "GTOJAF",
+      "GQSSAF",
+      // Teenage Mutant Ninja Turtles 2: Battle Nexus
+      "GNIEA4",
+      "GNIPA4",
+      // Teenage Mutant Ninja Turtles 3: Mutant Nightmare
+      "G3QEA4",
+      // The Lord of the Rings: The Third Age
+      "G3AP69",
+      "G3AF69",
+      "G3AE69",
+      "G3AJ13",
+      "G3AD69",
+      "G3AS69",
+      // Tiger Woods PGA Tour 2004
+      "GW4E69",
+      "GW4P69",
+      // Tiger Woods PGA Tour 2005
+      "G5TP69",
+      "G5TE69",
+      // Tom Clancy's Splinter Cell: Chaos Theory
+      "GCJE41",
+      "GCJP41",
+      // Tom Clancy's Splinter Cell: Double Agent (GC)
+      "GWYX41",
+      "GWYE41",
+  };
+  return std::find(two_disc_game_ids.begin(), two_disc_game_ids.end(), game_id) !=
+         two_disc_game_ids.end();
```

Comment:
If the list was sorted you could replace this with `std::binary_search`. Not that this is a particularly hot-path of code, but it's an easy optimization. :P